powerauth-backend-tests/src/test/java/com/wultra/security/powerauth/test/v31/PowerAuthIdentityVerificationTest.java

```diff
@@ -1042,9 +999,9 @@ private void submitDocuments(DocumentSubmitRequest submitRequest, List<FileSubmi
         assertNotNull(responseOtpOK.getEncryptedData());
         assertNotNull(responseOtpOK.getMac());
 
-        List<DocumentStatus> expectedStatuses = config.isVerificationOnSubmitEnabled() ?
-                ImmutableList.of(DocumentStatus.REJECTED, DocumentStatus.UPLOAD_IN_PROGRESS) : ImmutableList.of(DocumentStatus.VERIFICATION_PENDING);
-        boolean documentVerificationPending = false;
+        // TODO - check that this state is correct, I would expect that submitted document is in VERIFICATION_PENDING
```

Comment:
I will revisit this later. After https://github.com/wultra/enrollment-server/pull/514 this could be `ACCEPTED`, but will see.